src/prefect/engine/result_handlers/constant_result_handler.py

```diff
@@ -0,0 +1,38 @@
+from typing import Any
+
+from prefect.engine.result_handlers import ResultHandler
+
+
+class ConstantResultHandler(ResultHandler):
+    """
+    Hook for storing and retrieving constant Python objects. Only intended to be used
+    internally.
```

Comment:
What do you mean by _internally_?

Reply:
Only the "Prefect engine" should ever deal with this result handler, I don't recommend any users ever use it